21:44:17.129 Running build in Washington, D.C., USA (East) – iad1
21:44:17.134 Build machine configuration: 2 cores, 8 GB
21:44:17.380 Cloning github.com/Shriii19/FlowForge (Branch: optimize-analytics-data-preparation, Commit: b895a26)
21:44:17.817 Cloning completed: 436.000ms
21:44:17.954 Found .vercelignore
21:44:17.957 Removed 0 ignored files defined in .vercelignore
21:44:18.566 Restored build cache from previous deployment (8eCkBLc3Cd1QC5xBRwzYDv9Z1dj7)
21:44:19.060 Running "vercel build"
21:44:19.086 Vercel CLI 54.9.0
21:44:19.316 Running "install" command: `npm install --no-workspaces`...
21:44:26.153 
21:44:26.153 up to date, audited 453 packages in 7s
21:44:26.153 
21:44:26.153 153 packages are looking for funding
21:44:26.153   run `npm fund` for details
21:44:26.188 
21:44:26.188 5 moderate severity vulnerabilities
21:44:26.189 
21:44:26.189 To address issues that do not require attention, run:
21:44:26.190   npm audit fix
21:44:26.190 
21:44:26.190 To address all issues (including breaking changes), run:
21:44:26.191   npm audit fix --force
21:44:26.191 
21:44:26.191 Run `npm audit` for details.
21:44:26.232 Detected Next.js version: 16.2.6
21:44:26.233 Running "npm run build"
21:44:26.334 npm warn config ignoring workspace config at /vercel/path0/frontend/.npmrc
21:44:26.357 
21:44:26.357 > frontend@0.1.0 build
21:44:26.357 > cross-env NODE_OPTIONS=--max-old-space-size=4096 next build
21:44:26.358 
21:44:27.080   Applying modifyConfig from Vercel
21:44:27.095 ▲ Next.js 16.2.6 (Turbopack)
21:44:27.096 
21:44:27.099 ⚠ The "middleware" file convention is deprecated. Please use "proxy" instead. Learn more: https://nextjs.org/docs/messages/middleware-to-proxy
21:44:27.127   Creating an optimized production build ...
21:44:40.485 Turbopack build encountered 1 warnings:
21:44:40.486 ./frontend/tailwind.config.ts:13:13
21:44:40.486 Module not found: Can't resolve 'tailwindcss-animate'
21:44:40.486   11 |   },
21:44:40.486   12 |
21:44:40.487 > 13 |   plugins: [require("tailwindcss-animate")],
21:44:40.487      |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
21:44:40.487   14 | };
21:44:40.488   15 |
21:44:40.488   16 | export default config;
21:44:40.488 
21:44:40.488 
21:44:40.488 
21:44:40.488 https://nextjs.org/docs/messages/module-not-found
21:44:40.488 
21:44:40.488 
21:44:40.503 ✓ Compiled successfully in 13.1s
21:44:40.517   Running TypeScript ...
21:44:46.717 Failed to type check.
21:44:46.717 
21:44:46.718 ./app/components/analytics-comp/MemberAnalytics.tsx:31:28
21:44:46.718 Type error: Cannot find name 'useMemo'.
21:44:46.718 
21:44:46.718   29 |   onSelectMember,
21:44:46.718   30 | }: MemberAnalyticsProps) {
21:44:46.718 > 31 |   const analyticsMetrics = useMemo(
21:44:46.718      |                            ^
21:44:46.719   32 |     () => {
21:44:46.719   33 |       const completionRate =
21:44:46.719   34 |         getCompletionRate(
21:44:46.762 Next.js build worker exited with code: 1 and signal: null
21:44:46.799 npm error Lifecycle script `build` failed with error:
21:44:46.800 npm error code 1
21:44:46.800 npm error path /vercel/path0/frontend
21:44:46.800 npm error workspace frontend@0.1.0
21:44:46.800 npm error location /vercel/path0/frontend
21:44:46.800 npm error command failed
21:44:46.801 npm error command sh -c cross-env NODE_OPTIONS=--max-old-space-size=4096 next build
21:44:46.819 Error: Command "npm run build" exited with 1

--- a/frontend/app/components/analytics-comp/MemberAnalytics.tsx
+++ b/frontend/app/components/analytics-comp/MemberAnalytics.tsx
@@ -1,5 +1,5 @@
 "use client";
-
+import { useMemo } from "react";
 import type { MemberPerformance } from "./PerformanceSummary";
 
 type MemberAnalyticsProps = {

